src/Microsoft.DotNet.Build.Tasks/Targets/publishtest.targets

```diff
@@ -83,7 +83,7 @@
       OverwriteReadOnlyFiles="$(OverwriteReadOnlyFiles)"
       Retries="$(CopyRetryCount)"
       RetryDelayMilliseconds="$(CopyRetryDelayMilliseconds)"
-      UseHardlinksIfPossible="true">
+      UseHardlinksIfPossible="$(CreateHardLinksForCopyFilesToOutputDirectoryIfPossible)">
```

Comment:
Ugh, I'm having trouble testing that condition. When I build test projects with /p:OSGroup=Linux, it sets OS to Windows_NT (!?), and building with /p:OS=Linux fails outright (!?).